src/tree/SubscriptionTreeItem.ts

```diff
@@ -72,9 +72,12 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         ];
         LocationListStep.addStep(wizardContext, promptSteps);
 
+        const executeSteps: AzureWizardExecuteStep<IPostgresServerWizardContext | ICosmosDBWizardContext>[] = [];
+        executeSteps.push(new VerifyProvidersStep(['Microsoft.Web', 'Microsoft.Storage', 'Microsoft.Insights']));
```

Comment:
Those providers are for App Service. You have to pick the providers for your scenario, and you will have to handle the fact that Cosmos and Postgres probably have different providers that they need

Reply:
Didn't realize that 😅  Thanks for catching it!